source/dub/generators/generator.d

```diff
@@ -383,36 +383,30 @@ class ProjectGenerator
 		}
 
 		// 5. override string import files in dependencies
-		static void overrideStringImports(ref TargetInfo ti, TargetInfo[string] targets, string[] overrides)
+		static void overrideStringImports(ref TargetInfo ti, TargetInfo* pti, TargetInfo[string] targets, string[] overrides)
```

Comment:
I know it would be a bit verbose, but I'd rename `pti` to `parentTi` (or `parent_ti`, I'm not well-versed with vibe.d's style guide), so that it's clear further down that we're (lazily) pretending the dependent package's import files to those each dependency if we detect that we need to perform an override.

On second thought, if what I wrote above is correct, a comment along those lines would be quite helpful for other people looking at the code at a later time.